main.go

```diff
@@ -192,13 +194,25 @@ func keygenCmd(c *cli.Context) error {
 	if !args.Present() {
 		slog.Fatal("Missing drand address in argument (IPv4, dns)")
 	}
+
+	addr := args.First()
+	var validID = regexp.MustCompile(`[:][0-9]+$`)
+	if !validID.MatchString(addr) {
+		slog.Print("No port given. Please, choose a port number (or ENTER for default port 8080): ")
+		var port string
+		fmt.Scanf("%s\n", &port)
+		if port == "" {
+			port = default_port
+		}
+		addr = addr + ":" + string(port)
```

Comment:
Is there a need to cast to a string here ? 
If not, then please remove it (I'm pretty sure the Go compiler would optimize this away anyway but it's just cleaner without :) )